samples/net/mqtt_azure/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  description: MQTT sample app to Azure cloud
+  name: mqtt-azure
+tests:
+  sample.net.mqtt_azure:
+    harness: net
+    platform_whitelist: sam_e70_xplained frdm_k64f
+    tags: net mqtt
```

Comment:
add also `cloud` tag here, then it would be easier to just test the cloud specific stuff